db/hardware_test.go

```diff
@@ -211,6 +204,192 @@ func TestDeleteHardware(t *testing.T) {
 	}
 }
 
+type getFunc func(c context.Context, db *db.TinkDB, in *hardware.Hardware) (string, error)
+
+func TestGetHardware(t *testing.T) {
```

Comment:
You are testing different functions in the same test, please split the functions you are testing in their own test case (keep table testing as well) thanks